dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityNavigateEditMode.java

```diff
@@ -42,20 +40,17 @@
                 "window.top.document.dispatchEvent(customEvent);" +
             "</script>";
 
-    public VelocityNavigateEditMode(final HttpServletRequest request,
-                                    final HttpServletResponse response,
-                                    final String uri,
-                                    final Host host) {
-        super();
-        this.request = request;
-        this.response = response;
-        this.uri = uri;
-        this.host = host;
-        this.user = WebAPILocator.getUserWebAPI().getUser(request);
-    }
 
-    public VelocityNavigateEditMode(HttpServletRequest request, HttpServletResponse response) {
-        this(request, response, request.getRequestURI(), hostWebAPI.getCurrentHostNoThrow(request));
+    public VelocityNavigateEditMode(
```

Comment:
The constructor signature changed here too

Reply:
yeah you have to try to keep the backwards compatibility, since dotcms is a framework

Reply:
done https://github.com/dotCMS/core/pull/17859/commits/ec404552260c00230d3cce3ac019d69c6a6cea57#diff-8aaa91d048a2b9c6c8fac2ca7dc1c044R60